values.yaml

```
@@ -170,6 +170,7 @@ injector:
 
   # Injector service specific config
   service:
+    externalTrafficPolicy: "Cluster"
```

Quotes on the value are not needed:

```suggestion
    # Set the Service's externalTrafficPolicy
    # For the set of possible value see: https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
    externalTrafficPolicy: Cluster
```

---

Got it, I will look for the second instance of this value and update accordingly while removing this configuration under the injector service.